test/e2e/e2e_test.go

```diff
@@ -33,8 +33,8 @@ func TestE2E(t *testing.T) {
 	client, err := client.New(cfg, client.Options{})
 	require.NoError(t, err)
 
-	require.NoError(t, v1alpha2.AddToScheme(client.Scheme()))
-	require.NoError(t, v1beta1.AddToScheme(client.Scheme()))
+	require.NoError(t, gwapiv1a2.AddToScheme(client.Scheme()))
```

Comment:
why keep this?

Reply:
For TCPRoute, GRPCRoute, TLSRoute, UDPRoute